Gemfile.lock

```diff
@@ -148,7 +148,7 @@ DEPENDENCIES
   tzinfo-data
 
 RUBY VERSION
-   ruby 2.6.0p0
+   ruby 2.6.3p62
```

Comment:
Why is this needed?

Reply:
It is my local development environment. You can ignore this change.